content/en/docs/reference/config/proxy_extensions/wasm-plugin/index.html

```diff
@@ -7,7 +7,7 @@
 layout: protoc-gen-docs
 generator: protoc-gen-docs
 schema: istio.extensions.v1alpha1.WasmPlugin
-aliases: [/docs/reference/config/extensions/v1alpha1/wasm-plugin]
+aliases: [/docs/reference/config/proxy_extensions/wasm-plugin/]
```

Comment:
This change cannot be made in this file. Note the file header: WARNING: THIS IS AN AUTO-GENERATED FILE, DO NOT EDIT. PLEASE MODIFY THE ORIGINAL SOURCE IN THE 'https://github.com/istio/api' REPO. 

This file is pulled from the istio/api repo. You would have to change the alias there.

Reply:
https://github.com/istio/api/blob/master/extensions/v1alpha1/wasm.proto#LL25C1-L25C70

Reply:
That said, I think that link has been the same for a long time.

Reply:
Note that you are aliasing the doc to its own address with your change, so the alias is simply being removed in this code change, and a new one is not being generated.

Reply:
> This change cannot be made in this file. Note the file header: WARNING: THIS IS AN AUTO-GENERATED FILE, DO NOT EDIT. PLEASE MODIFY THE ORIGINAL SOURCE IN THE 'https://github.com/istio/api' REPO.
> 
> This file is pulled from the istio/api repo. You would have to change the alias there.

Thanks  @ericvn for the above information, I will take care of it in the future. But yesterday when I tried it was not working for me, maybe the changes did in the PR: #143 were not reflected yesterday. but now, it is working for me also.

Reply:
Also, I have one concern,the following link which is mentioned under the alias is not working, is it intentional or  is it a bug or did I miss something here.
https://istio.io/latest/docs/reference/config/extensions/v1alpha1/wasm-plugin

Reply:
I will close this PR and the corresponding issue if there is no bug.
Thanks

Reply:
> Also, I have one concern,the following link which is mentioned under the alias is not working, is it intentional or is it a bug or did I miss something here. https://istio.io/latest/docs/reference/config/extensions/v1alpha1/wasm-plugin

Hi, @ericvn , your thoughts on above comment?
PTAL!